content/developers/overview/github-developer-program.md

```diff
@@ -28,7 +28,7 @@ Email <a href="mailto:partnerships@github.com">partnerships@github.com</a> to re
 ## Have an integration that works with GitHub?
 
 Awesome! We'd love to have you be part of the program. Here’s how you can spread the word:</p>
-* [Let us know about your integration](https://github.com/contact?form[subject]=New+GitHub+Integration)
+* [Let us know about your integration](https://support.github.com/contact?tags=rr-general-technical)
```

Comment:
```suggestion
* [Let us know about your integration](https://support.github.com/contact?tags=rr-general-technical&form[subject]=New+GitHub+Integration)
```

A small addition to autofill the subject line with `New GitHub Integration` and keep parity with the original link.

Reply:
Ahh that's the addition needed to autopopulate subject, couldn't quite figure it out when just trying to add data from the old link.